pkg/testutil/nerdtest/requirements.go

```diff
@@ -416,3 +417,15 @@ var RemapIDs = &test.Requirement{
 		return false, "snapshotter does not support ID remapping"
 	},
 }
+
+var ContainerdVersionV2 = &test.Requirement{
+	Check: func(data test.Data, helpers test.Helpers) (ret bool, mess string) {
+		stdout := helpers.Capture("info", "--format", "{{ json . }}")
+		var dinf dockercompat.Info
+		err := json.Unmarshal([]byte(stdout), &dinf)
+		assert.NilError(helpers.T(), err, "failed to parse docker info")
+		matched, err := regexp.MatchString(`^v2\.\d+`, dinf.ServerVersion)
```

Comment:
semver should be used
https://github.com/containerd/nerdctl/blob/772bb09ba518dd0ba855d0dc0ddb07aa154c59c1/pkg/taskutil/taskutil.go#L144

Reply:
resove